package.json

```diff
@@ -184,13 +184,16 @@
     "source-map-fixtures": "^2.1.0",
     "tap": "^7.1.2",
     "touch": "^1.0.0",
+    "ts-node": "^1.6.0",
+    "typescript": "^2.0.3",
     "xo": "^0.17.0",
     "zen-observable": "^0.3.0"
   },
   "xo": {
     "rules": {
       "import/newline-after-import": "off",
-      "no-use-extend-native/no-use-extend-native": "off"
+      "no-use-extend-native/no-use-extend-native": "off",
+      "no-useless-escape": "off"
```

Comment:
Ok. fixed.

but this part looks to be a little weird:

```js
  var isBabelPath = /^babel-runtime[\\/]?/.test(path.node.value);
```

Ref: http://stackoverflow.com/a/40315390/1123955

And a question: How did the AVA pass the CI test? I just clone the AVA repo, run `npm install && npm t`, XO will throw `no-useless-escape` error.